patches/unified-engine+11.1.1.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/node_modules/unified-engine/index.d.ts b/node_modules/unified-engine/index.d.ts
+index 0e25d67..503f1ab 100644
+--- a/node_modules/unified-engine/index.d.ts
++++ b/node_modules/unified-engine/index.d.ts
+@@ -1,4 +1,4 @@
+-export { Configuration } from "./lib/configuration.js";
++export { Configuration, Result } from "./lib/configuration.js";
```

Comment:
@wooorm May I raise a PR to export type `Result`?

Reply:
Sure. I’d prefer it renamed in `unified-engine` to `ConfigurationResult` then though!

Reply:
See https://github.com/unifiedjs/unified-engine/pull/76